docs/src/tutorials/qless.md

```diff
@@ -0,0 +1,58 @@
+```@example qless1
+using QRMumps, LinearAlgebra, SparseArrays, Printf
+
+irn = [1, 3, 5, 2, 3, 5, 1, 4, 4, 5, 2, 1, 3]
+jcn = [1, 1, 1, 2, 3, 3, 4, 4, 5, 5, 6, 7, 7]
+val = [1.0, 2.0, 3.0, 1.0, 1.0, 2.0, 4.0, 1.0, 5.0, 1.0, 3.0, 6.0, 1.0]
+
+A = sparse(irn, jcn, val, 5, 7)
+b = [40.0, 10.0, 44.0, 98.0, 87.0]
+y_star = [0.0, 1.0, 2.0, 3.0, 4.0]
+y₁ = similar(b,7)
+y₂ = similar(b)
+
+qrm_init()
+
+spmat = qrm_spmat_init(A)
```

Comment:
Please add a comment next to each QRMumps call to explain what it does:

- initialize data structure
- perform symbolic analysis
- etc.

That is very useful when a user only looks at an example.

Reply:
I did like in other files of this folder I can adapt this but they should be changed too then

Reply:
I understand. We can change the other examples separetely.

Reply:
@MaxenceGollier ⬆️